subprojects/docs/src/docs/userguide/running-builds/command_line_interface.adoc

```diff
@@ -15,247 +15,282 @@
 [[command_line_interface]]
 = Command-Line Interface
 
-[.lead]
-The command-line interface is one of the primary methods of interacting with Gradle. The following serves as a reference of executing and customizing Gradle use of a command-line or when writing scripts or configuring continuous integration.
+The command-line interface is the **primary method of interacting with Gradle**.
 
-Use of the <<gradle_wrapper.adoc#gradle_wrapper, Gradle Wrapper>> is highly encouraged. You should substitute `./gradlew` or `gradlew.bat` for `gradle` in all following examples when using the Wrapper.
+The following is a reference for executing and customizing the Gradle command-line.
+It also serves as a reference when writing scripts or configuring continuous integration.
 
-Executing Gradle on the command-line conforms to the following structure. Options are allowed before and after task names.
+**Use of the <<gradle_wrapper.adoc#gradle_wrapper, Gradle Wrapper>> is highly encouraged**.
+Substitute `./gradlew` or `gradlew.bat` for `gradle` in the following examples.
```

Comment:
I'm wondering if it would be better to use ./gradlew in the examples and mention here to substitute with gradlew.bat for Windows

Reply:
I updated this sentence so its a little more clear about which to use.